test/memkind_memtier_data_movement_test.cpp

```diff
@@ -1604,6 +1636,141 @@ TEST_F(RankingPerfTest, PageMovement)
     }
 
     // Measure access times on DRAM
-    auto dram_time2 = bench(m_bench_num, m_alloc_size, m_start_addr);
+    auto dram_time2 = bench(bench_num, m_alloc_size, m_start_addr);
     ASSERT_TRUE(dram_time2 * 2 < pmem_time);
 }
+
+TEST_F(RankingPerfTest, PEBS)
```

Comment:
This function has around 130 lines of code and contains some code repetitions, e.g. 
1. Bench pmem
2. Wait
3, Print times
is repeated a few times.

Maybe we could use a ```for``` loop ? Or create a function to encapsulate some of the code that is repeated?